rsr_convert_scripts/fci_rsr.py

```diff
@@ -22,63 +22,52 @@
 
 """Read the MTG FCI relative spectral response functions.
 
-Data from EUMETSAT NWP-SAF:
-https://nwpsaf.eu/downloads/rtcoef_rttov12/ir_srf/rtcoef_mtg_1_fci_srf.html
+Data from EUMETSAT:
+https://sftp.eumetsat.int/public/folder/UsCVknVOOkSyCdgpMimJNQ/User-Materials/MTGUP/Materials/FCI-SRF_Apr2022/
 """
 
-import os
 import logging
-import numpy as np
+from netCDF4 import Dataset
 from pyspectral.utils import convert2hdf5 as tohdf5
 from pyspectral.raw_reader import InstrumentRSR
 
 LOG = logging.getLogger(__name__)
 
-FCI_BAND_NAMES = ['ch1', 'ch2', 'ch3', 'ch4', 'ch5', 'ch6', 'ch7', 'ch8', 'ch9', 'ch10',
-                  'ch11', 'ch12', 'ch13', 'ch14', 'ch15', 'ch16']
+FCI_BAND_NAMES = ['VIS0.4', 'VIS0.5', 'VIS0.6_HR', 'VIS0.8', 'VIS0.9', 'NIR1.3',
```

Comment:
Would it be possible to use `pyspectral.bandnames.BANDNAMES['fci'].values()` here? That would a void duplicating the list.

Reply:
Indeed yes, thanks. Much better.